docs/DEPLOYMENT.md

```diff
@@ -146,22 +146,68 @@ Follow the steps below to set up the staging environment on AWS.
+         7. Select **Advanced Redis settings** and fill in the details as mentioned below:
+            1. For *Subnet group* you can create a new group or use an existing one. For staging, subnets were created in step `2.vii`. You can create a new subnet group, name it accordingly and add the subnets (created in step `2.vii`) to that group. A VPC was also created in step `2`. Add that VPC ID in the appropriate selectbox. *NOTE*: If you selected Multi-AZ checkbox in step `11.iv.f`, then the subnet group will require two or more subnets in the group. For example, in staging environment for Plio, three subnets have been created and added to the group.
+            2. For *Availability zone(s)*, choose *No preference* – ElastiCache chooses the Availability Zones for your cluster's nodes. If required, configure your own availability zones. For more info, see [this](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/RegionsAndAZs.html)
+            3. For Security groups, choose the security groups that you want for this cluster. A security group acts as a firewall to control network access to your cluster. You can choose use the Default security group for your VPC or create a new one.
+            4. (OPTIONAL) Configure encryption options according to your requirements.
+            5. (OPTIONAL) Configure logging. For Plio, JSON logging has been configured at CloudWatch Logs
+            6. (OPTIONAL) Rest all the options are not mandatory and are left up to the user to configure according to their needs
+         8. Click the **Create** button. A Redis Cluster will get initialized and once it becomes `available` you will be able to continue with the steps ahead.
+         9. Grant access to your redis cluster by following the steps below-
+            1. Go to the AWS EC2 console [here](https://console.aws.amazon.com/ec2/)
+            2. In the navigation pane, under **Network & Security**, choose **Security Groups**.
+            3. From the list of security groups, choose the security group for your Amazon VPC. Unless you created a security group for ElastiCache use, this security group will be named **default**.
+            4. Choose the Inbound tab, and then do the following:
+               1. Choose **Edit** -> **Add Rule**.
+               2. In the **Type** column, choose **Custom TCP rule**.
+               3. In the **Port range** box, type the port number for your cluster node. This number must be the same one that you specified when you launched the cluster. The default port for Redis is `6379`.
+               4. In the **Source** box, choose Anywhere which has the port range `(0.0.0.0/0)` so that any Amazon EC2 instance that you launch within your Amazon VPC can connect to your ElastiCache nodes.
```

Comment:
not clear.
```suggestion
               4. In the **Source** box, choose Anywhere which has the port range `(0.0.0.0/0)` so that ant EC2 instance that you launch within your VPC can connect to your ElastiCache nodes.
```